src/Appwrite/Utopia/Response/Model/Deployment.php

```
@@ -190,6 +191,12 @@ public function getName(): string
         return 'Deployment';
     }
 
+    public function filter(Document $document): Document
+    {
+        $this->applySelectQueries($document, $request);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$request'. (Unused Code Rules)

(UnusedLocalVariable)